src/lib/components/Communities/search/CommunityCompactItemComputer.js

```diff
@@ -39,19 +39,18 @@ export const CommunityCompactItemComputer = ({
                 {metadata.title}
               </a>
             </Item.Header>
-
+            <div className="mb-5">
```

Comment:
why was it moved here? I refactored it lately to follow the correct samntic ui implementation using Item.Extra

Reply:
I did this because if we keep the labels in the `Item.Extra` they overlap with the description. 
## Before
![Screenshot from 2023-03-29 15-55-00](https://user-images.githubusercontent.com/15194802/228561129-ac65bf76-bf0f-483b-9775-15aae2d7bc3e.png)

## Now
![Screenshot from 2023-03-29 15-53-54](https://user-images.githubusercontent.com/15194802/228561199-d0876127-c6e2-4199-bdab-4c7e4877608a.png)